### PR TITLE
fix: adjust components accordingly issues #85 #84 #83 -

### PR DESCRIPTION
**Pull request by horberlan**
# Pull request template

## Description

fix some issues.

Fixes #(issue)
#84 
#83 
#85 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Vercel Preview deployment has succeeded
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings
- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) document

### CODE REVIEW

**Comment by horberlan** [comment]
@Looskie can you rerun pipelines?

**Comment by Looskie** [comment]
> @Looskie can you rerun pipelines?

They failed because of an eslint error on https://github.com/dahliaOS/website/pull/86/files#diff-4f34a078049f572f5ebcdf8a3b771ce47c64ec9808cf3acf95a04e1ab72c214dR201

We have our logs public, so you can access them here. https://vercel.com/dahliaos/website/AhZyjaBsZCr3etUBasKBL5Bwf2AK

Could you fix the linting issue? Otherwise, I can.

**Comment by horberlan** [comment]
Done. In the next contribution I intend to internationalize the platform with i18n.

File: components/Footer.tsx [hunk 1]
```diff
@@ -198,6 +200,7 @@ type ThemeTypes = "dark" | "light" | "system";
 const Footer = () => {
   const [localStorageTheme, setLocalStorageTheme] = useState<ThemeTypes>();
   const theme = useTheme();
+  const preferredTheme = usePreferredTheme() || localStorageTheme;
```
Comment (Looskie) [inline components/Footer.tsx:203]: tbh, we should just remove the useState for localStorageTheme, I wrote this hook a bit ago so sorry if im not remembering correctly, but `usePreferredTheme` listens for storage events which renders the state kinda useless.

**Review by Looskie — CHANGES_REQUESTED** [review]
other than the small comment above, looks good! many thanks for your contribution ❤️

File: components/Footer.tsx (hunk 1 above)
Comment (horberlan) [reply components/Footer.tsx]: is it what you needed? At Front I'm a vue.js developer, react syntax sounds a little different...

Comment (Looskie) [reply components/Footer.tsx]: bingo! exactly what i mean. looks good! thank you!

**Pull request merged**